ArcGISRuntimeSDKQt_CppSamples/Layers/TileCacheLayer/TileCacheLayer.cpp

```diff
@@ -26,6 +26,7 @@
 #include "TileCache.h"
 
 #include <QDir>
+#include <QStandardPaths>
```

Comment:
Ideally this would be in an `Q_OS_IOS` ifdef, but you could make the same argument for QDir as well (only for Android).

This is good to keep the code tidy.